src/js/pages/ClusterPage.js

```diff
@@ -8,7 +7,7 @@ class ClusterPage extends React.Component {
   render() {
     return (
       <Page title="Cluster">
-        <RouteHandler />
+        {this.props.children}
```

Comment:
Are these an intermediate step to upgrading?
I really think these should remain routes